app/scripts/disable-console.js

```diff
@@ -0,0 +1,6 @@
+// disable console.log in contentscript to prevent SES/lockdown logging to external page
+// eslint-disable-next-line import/unambiguous
+if (typeof window.console !== undefined) {
```

Comment:
I wish we could leave this on in development at least :thinking:

Reply:
I was thinking that we wanted to avoid inserting any products of our build process before the lockdown, but we already have `initSentry` running first for some contexts. So maybe injecting the environment here using `loose-envify` would be OK.